ovos_config/mycroft.conf

```diff
@@ -696,6 +696,12 @@
         "active": true
       }
     },
+    // audio service ducking - lowering volume of background audio when TTS or system sound 
+    // is playing
+    // NOTE: If the TTS config 'pulse_duck' is set to true, this setting will be ignored
+    // and pulse audio role ducking will be used instead.
+    "duck": true,
```

Comment:
can we leave this as False by default for now? Kinda want the OCP sprint to be done before we change this, as i am introducing both "duck" and "cork" in there   (volume change vs pause/unpause)